tensorflow/lite/micro/tools/python_interpreter/src/interpreter_wrapper.cc

```diff
@@ -0,0 +1,223 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
```

Comment:
For reviewers, the code here is mostly a lightweight version of https://github.com/tensorflow/tensorflow/tree/v2.9.0/tensorflow/lite/python/interpreter_wrapper reworked to be compatible with TFLM.